sdk/storage/Azure.Storage.Blobs/src/AssemblyInfo.cs

```diff
@@ -16,3 +16,15 @@
     "012ea67d2479751a0b8c087a4185541b851bd8b16f8d91b840e51b1cb0ba6fe647997e57429265" +
     "e85ef62d565db50a69ae1647d54d7bd855e4db3d8a91510e5bcbd0edfbbecaa20a7bd9ae74593d" +
     "aa7b11b4")]
+[assembly: InternalsVisibleTo("Azure.Storage.Files.DataLake, PublicKey=" +
+    "0024000004800000940000000602000000240000525341310004000001000100097AD52ABBEAA2" +
+    "E1A1982747CC0106534F65CFEA6707EAED696A3A63DAEA80DE2512746801A7E47F88E7781E71AF" +
+    "960D89BA2E25561F70B0E2DBC93319E0AF1961A719CCF5A4D28709B2B57A5D29B7C09DC8D269A4" +
+    "90EBE2651C4B6E6738C27C5FB2C02469FE9757F0A3479AC310D6588A50A28D7DD431B907FD325E" +
+    "18B9E8ED")]
+[assembly: InternalsVisibleTo("Azure.Storage.Files.DataLake.Tests, PublicKey=" +
```

Comment:
There's a chance you might need one of these in Common too if you need any of its internals.

Reply:
fixed.